README.md

```diff
@@ -860,6 +860,92 @@ Promise.all(require("mobile-cli-lib").liveSyncService.deleteFiles(deviceInfos, p
 	});
 ```
 
+### Module typeScriptService
+> Stability: 1 - Could be changed due to some new requirments.
+
+This module is used to transpile TypeScript files.
+
+The following types are used:
+```TypeScript
+interface ITypeScriptCompilerOptions {
+	codePage?: number; // Specify the codepage to use when opening source files.
+	declaration?: boolean; //  Generates corresponding .d.ts file.
+	mapRoot?: string; //  Specifies the location where debugger should locate map files instead of generated locations.
+	module?: string; // Specify module code generation: 'commonjs' or 'amd'.
+	noImplicitAny?: boolean; //  Warn on expressions and declarations with an implied 'any' type.
+	outFile?: string; // Concatenate and emit output to single file.
+	outDir?: string; // Redirect output structure to the directory.
+	removeComments?: boolean; // Do not emit comments to output.
+	sourceMap?: boolean; // Generates corresponding .map file
+	sourceRoot?: string; // Specifies the location where debugger should locate TypeScript files instead of source locations.
+	target?: string;  // Specify ECMAScript target version: 'ES3' (default), or 'ES5'.
+	noEmitOnError?: boolean; // Do not emit outputs if any errors were reported.
+	[key: string]: any;
+}
+
+/**
+ * Describes the options for transpiling TypeScript files.
+ */
+interface ITypeScriptTranspileOptions {
+	/**
+	 * Describes the options in tsconfig.json file.
+	 */
+	compilerOptions?: ITypeScriptCompilerOptions;
+
+	/**
+	 * The default options which will be used if there is no tsconfig.json file.
+	 */
+	defaultCompilerOptions?: ITypeScriptCompilerOptions;
+
+	/**
+	 * Path to the default .d.ts files.
+	 */
+	pathToDefaultDefinitionFiles?: string;
+}
+```
+
+* `transpile(projectDir: string, typeScriptFiles?: string[], definitionFiles?: string[], options?: ITypeScriptTranspileOptions): Promise<string>` - Transpiles specified files or all files in the project directory.
+The default passed options are overriden by the ones in tsconfig.json file.
```

Comment:
This sentence along with the one below it are a bit confusing IMO because both sentences refer to `passed` options, however they mean different things. It could be something more like the description of this PR:
```
If `options` are not specified the method will search for tsconfig.json file and get them from it. If there is no tsconfig.json file the method will use default options.
```